frontend/vuejs/src/views/Subjects/AddCreateSubject.vue

```diff
@@ -3,138 +3,153 @@
+              <div style="margin-left: 10%" class="radio">
+                <label
+                  ><input
+                    v-if="Subject.is_active == false"
+                    type="radio"
+                    id="an"
+                    name="exampleRadios"
+                    value=""
+                    checked
+                  />
+                  <input
+                    v-else
+                    type="radio"
+                    id="an"
+                    name="exampleRadios"
+                    value=""
+                  />{{ $t("list_subjects.label.unActive") }}</label
+                >
+              </div>
+              <div style="margin-left: 10%" class="radio">
+                <label
+                  ><input
+                    v-if="Subject.is_active"
+                    type="radio"
+                    id="hien"
+                    name="exampleRadios"
+                    value=""
+                    checked
+                  />
+                  <input
+                    v-else
+                    type="radio"
+                    id="hien"
+                    name="exampleRadios"
+                    value=""
+                  />{{ $t("list_subjects.label.active") }}</label
+                >
+              </div>
+              <div
+                class="form-group"
+                style="margin-left: 10%; margin-right: 10%"
+              >
+                <label for="exampleInputPassword1">{{
+                  $t("list_subjects.label.course")
+                }}</label>
+                <multiselect
+                  v-model="value"
+                  :options="courseData"
+                  :multiple="true"
+                  :close-on-select="false"
+                  :clear-on-select="false"
+                  :preserve-search="true"
+                  :placeholder="$t('list_subjects.label.nameCourse')"
+                  label="name"
+                  track-by="name"
+                  :preselect-first="true"
+                >
+                </multiselect>
+              </div>
+
+              <button
+                v-if="id"
+                type="submit"
+                class="btn btn-success"
+                style="float: right; margin-right: 10%"
+                v-on:click="update()"
+              >
+                {{ $t("list_subjects.button.update") }}
+              </button>
+              <button
+                v-else
+                type="submit"
+                class="btn btn-success"
+                style="float: right; margin-right: 10%"
+                v-on:click="add()"
+              >
+                {{ $t("list_subjects.button.add") }}
+              </button>
+              <router-link
+                style="float: right; background-color: red; margin-right: 2%"
+                class="btn btn-success"
+                v-bind:to="'/listSubjects/'"
+                >{{ $t("list_subjects.button.cancel") }}</router-link
+              >
```

Comment:
![image](https://user-images.githubusercontent.com/67667833/99748413-13b40200-2b0f-11eb-9425-d88c5edc5634.png)
v-bind:to =? :to
dùng router name thay cho router path